src/external-types/reveal.js/index.d.ts

```diff
@@ -4,9 +4,199 @@
+    fragments?: boolean,
+    // Add the current slide number to the URL hash so that reloading the
+    // page/copying the URL will return you to the same slide
+    /* hash?: boolean, // ToDo: I think this is not needed */
+    // Use 1 based indexing for # links to match slide number (default is zero
+    // based)
+    /* hashOneBasedIndex?: boolean, // ToDo: I think this is not needed */
+    // Flags if we should show a help overlay when the question-mark
+    // key is pressed
+    help?: boolean,
+    // Time before the cursor is hidden (in ms)
+    hideCursorTime?: number,
+    // Hide cursor if inactive
+    hideInactiveCursor?: boolean,
+    // Push each slide change to the browser history.  Implies `hash: true`
+    /* history?: boolean, // ToDo: I think this is not needed */
+    // Enable keyboard shortcuts for navigation
+    keyboard?: boolean,
+    // Optional function that blocks keyboard events when retuning false
+    //
+    // If you set this to 'foucsed', we will only capture keyboard events
+    // for embdedded decks when they are in focus
+    /* keyboardCondition: null, // ToDo: I think this is not needed */
+    // Loop the presentation
+    loop?: boolean,
+    // Number of slides away from the current that are visible on mobile
+    // devices. It is advisable to set this to a lower number than
+    // viewDistance in order to save resources
+    mobileViewDistance?: number,
+    // Enable slide navigation via mouse wheel
+    mouseWheel?: boolean,
+    // Changes the behavior of our navigation directions.
+    //
+    // "default"
+    // Left/right arrow keys step between horizontal slides, up/down
+    // arrow keys step between vertical slides. Space key steps through
+    // all slides (both horizontal and vertical).
+    //
+    // "linear"
+    // Removes the up/down arrows. Left/right arrows step through all
+    // slides (both horizontal and vertical).
+    //
+    // "grid"
+    // When this is enabled, stepping left/right from a vertical stack
+    // to an adjacent vertical stack will land you at the same vertical
+    // index.
+    //
+    // Consider a deck with six slides ordered in two vertical stacks:
+    // 1.1    2.1
+    // 1.2    2.2
+    // 1.3    2.3
+    //
+    // If you're on slide 1.3 and navigate right, you will normally move
+    // from 1.3 -> 2.1. If "grid" is used, the same navigation takes you
+    // from 1.3 -> 2.3.
+    navigationMode?: NavigationMode,
+    // Enable the slide overview mode
+    overview?: boolean,
+    // Flags if it should be possible to pause the presentation (blackout)
+    pause?: boolean,
+    // The maximum number of pages a single slide can expand onto when printing
+    // to PDF, unlimited by default
+    /* pdfMaxPagesPerSlide?: number, // ToDo: I think this is not needed */
+    /* pdfPageHeightOffset: -1, // ToDo: I think this is not needed */
+    /* pdfSeparateFragments: true, // ToDo: I think this is not needed */
```

Comment:
We don't include PDF printing so these are unnecessary, aren't they?

Reply:
It seems all reveal does is provide a custom stylesheet, to print to PDF. The presentation. Maybe we want to keep this around then?

Additional context: https://revealjs.com/pdf-export/

Reply:
We already have this in HedgeDoc 1.x (append `?print-pdf` to a /p/ URL). We should keep it.